tests/components/utility_meter/test_sensor.py

```diff
@@ -1654,6 +1664,10 @@ async def _test_self_reset(
             state.attributes.get("last_reset") == dt_util.as_utc(now).isoformat()
         )  # last_reset is kept in UTC
         assert state.state == "3"
+        # In first event state should be 0, issue #142053
```

Comment:
```suggestion
        # In first event state should be 0
```
I think the comment is clear even without having the issue referenced